### ISSUE TITLE
Hide Badges on camera/streaming control panel fails with Custom Namebadges

### ISSUE THREAD

**Issue by ProbablePrime** [Neos-Metaverse/NeosPublic#3725]
### Describe the bug?

Hide Badges on camera/streaming control panel fails with Custom Namebadges

### Relevant issues

Based on discussion in #3724 

### To Reproduce

1. Make a custom name badge(https://youtu.be/1CI71v6xFwQ)
2. Open Camera/Streaming control UI
3. Check Hide Badges
4. Observer your badge doesn't hide	

### Expected behavior

Badges to hide

### Log Files

N/A

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2022.1.28.1335 

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

### Additional context

From a quick review of the code, it looks like the code was never written to handle custom name badges. We should probably hook into the AvatarNamebadgeVisibilityDriver's functionality somehow.

### Reporters

[CoffeeKei](https://github.com/CoffeeKei)
EpicEaston197

**Comment by Lexevolution** [comment]
Yeah probably instead of just checking the autoNameBadge fields in the AvatarManager and excluding that from the camera render, just grab the slot reference from the AvatarNameplateVisibilityDriver and add that to the camera's render exclusion list.

Could probably do it in a similar way Neos checks for the AvatarNameTagAssigner already on an avatar, either through the AvatarManager or CommonAvatarBuilder. By finding the AvatarNameplateVisibilityDriver component in the child slots of the avatar, since there really should only be one of those components on an avatar.
